front_end/html/exercise.html

```diff
@@ -282,15 +282,11 @@ <h6 style="display:inline">Using Pair Programming:</h6>
                 </label><br />
 
                 <div id="invalid_partner"></div>
-                <div style="margin: 15px 0;" class="select is-grey space_div" >
-                    <select onchange="showUsers(this.value)" id="pair_programming" name="pair_programming" class="edit-select">
-                        <option value="Yes">Yes</option>
-                        <option value="No" selected>No</option>
-                    </select>
-                </div>
 
```

Comment:
Removes select input for pair programming, asks user to leave blank if not using pair programming instead